br/pkg/task/backup.go

```diff
@@ -296,7 +328,11 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	// Domain loads all table info into memory. By skipping Domain, we save
 	// lots of memory (about 500MB for 40K 40 fields YCSB tables).
 	needDomain := !skipStats
-	mgr, err := NewMgr(ctx, g, cfg.PD, cfg.TLS, GetKeepalive(&cfg.Config), cfg.CheckRequirements, needDomain, conn.NormalVersionChecker)
+	versionCheckerNum := conn.NormalVersionChecker
+	if cfg.UseCheckpoint {
```

Comment:
it seems we lost some check when using checkpoint?

Reply:
done. change to mark the support error in version pkg.